dpypelines/pipeline/README.md

```diff
@@ -4,33 +4,58 @@ Instructions on running existing pipelines can be found below. As more pipeline
 
 ## dataset_ingress_v1
 
-### Default configuration
+### Running locally
 
-The `dataset_ingress_v1` pipeline function can be run directly with a local directory of files using the default configuration details.
+The `dataset_ingress_v1` pipeline function can be run directly via a script like the following:
 
+`myscript.py`
 ```python
 from dpypelines.pipeline.configuration import get_pipeline_config
 from dpypelines.pipeline.dataset_ingress_v1 import dataset_ingress_v1
 
-# `files_dir` should be a string of the directory containing a data.xml file
-files_dir = "path/to/data/file"
+# `files_dir` should be a string of the directory containing your data source(s)
+files_dir = "path/to/directory/of/inputs"
 
-# Get the default configuration details
-default_pipeline_config, _ = get_pipeline_config("default")
+# Specify the dataset id for picking the configuration you want to use for
+# your source.
+pipeline_config = get_pipeline_config("<dataset_id>")
 
-dataset_ingress_v1(files_dir, default_pipeline_config)
+# Run the function
+dataset_ingress_v1(files_dir, pipeline_config)
 ```
 
-Running `dataset_ingress_v1` will generate three files:
-- a copy of the source data.xml file.
+Which you would run via:
+
+```
+poetry run python3 ./myscript.py
+```
+
+Note: the key you choose should match at least one key in the `CONFIGURATION` dict, see [this file](https://github.com/ONSdigital/dp-data-pipelines/blob/sandbox/dpypelines/pipeline/configuration.py).
+
+**Before** running this, you'll need to set some environment variables based on one of two scenarios as follows:
+
+### Scenario 1: You dont care about uploading the results of the pipeline (you just want to run the transform and see the outputs)
+
+This is probably the setup you'll want to use the majority of the time when developing locally.
+
+Set the following env vars:
+
+```
+export DISABLE_NOTIFICATIONS=true
```

Comment:
Need to set UPLOAD_SERVICE_URL as well (can be a nonsense string if not actually uploading)